ui/src/app/views/workspace/component.ts

```diff
@@ -8,6 +9,29 @@ import {RepositoryService} from 'app/services/repository.service';
 import {User} from 'app/models/user';
 import {UserService} from 'app/services/user.service';
 
+class Notebook {
+  constructor(public name: string, public description: string, public url: string) {}
+}
+class CohortNameFilter implements StringFilter<Cohort> {
```

Comment:
@markfickett This class seems short-enough that the purpose is succinctly explained by the code itself. Is there an additional benefit to a class doc of which I'm not aware?